Orchestrator.py

```diff
@@ -166,15 +290,32 @@ def getEvaluatedPositions(self):
         ]
         return returnVal
 
+    def _hCell2SvgRect(self,cx,cy):
+        returnVal = {
+            'x':        cx,
+            'y':        cy,
+            'width':    self.MINFEATURESIZE/2,
+            'height':   self.MINFEATURESIZE/2,
+        }
+        return returnVal
+
+    def getExploredCells(self):
+        returnVal = {
+                'cellsOpen':     [self._hCell2SvgRect(*c) for c in self.exploredCells],
+                'cellsObstacle': [self._hCell2SvgRect(*c) for c in self.obstacleCells],
+            }
+        return returnVal
+
+
     def getView(self):
         '''
         Retrieves the approximate location of the DotBot for visualization.
         '''
 
         returnVal = {
-            'dotbotpositions':    self.getEvaluatedPositions(),
-            'discomap':           None,
-            'exploredCells':      None,
+            'dotbotpositions': self.getEvaluatedPositions(),
+            'discomap': {"complete": False, "dots": [], "lines": []},
+            'exploredCells': self.getExploredCells(),
```

Comment:
are you calling `.getExploredCells()` from anywhere else? If not:
- just have two entries "exploredCells" and "obstacleCells" in your returnVal
- populate the same way it is now in `getExploredCells`
- delete `getExploredCells`